quack_app/test/core/notification_data_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quack_app/core/notification_data.dart';
+
+void main() {
+  group("MenuData", () {
+    NotificationData notificationData = NotificationData();
+    test("getExistingNotifications()", () async {
+      await notificationData.getExistingNotifications();
+      expect(notificationData.currentNotifications.length, 3);
```

Comment:
This will mean that the mocked notifications class in the future will need to return 3. Consider testing that the notifications list is not empty rather than a specific length